news/2018-08-12-project-loved-week-of-august-12th.md

```diff
@@ -0,0 +1,194 @@
+
+[DragonForce - ULTRA BEATDOWN SUPREME](https://osu.ppy.sh/beatmapsets/728851) by [IcyWorld](https://osu.ppy.sh/users/1383951)  
+*written by [Pope Gadget](https://osu.ppy.sh/users/2288341)*
+
+Holy heck, Icy made something spicy! This is a renowned Stepmania classic made more than seven years ago, but it still holds up interest today—a massive amalgamation of several DragonForce tracks charted in Icy's typical fun fashion. This was originally brought to the mania community as a convert back in 2014 from [Pichu](https://osu.ppy.sh/users/2521046) (which has since garnered over 30 favourites), but Icy has been so kind as to upload this onto his own account, such that the no converts rule could be overridden. Enjoy!
+
+[![](/wiki/shared/news/2018-08-12-project-loved-week-of-august-12th/mania/euclid.jpg)](https://osu.ppy.sh/community/forums/topics/788688)
+
+[Function Phantom - Euclid](https://osu.ppy.sh/beatmapsets/658097) by [juankristal](https://osu.ppy.sh/users/443656) and [XeoStyle](https://osu.ppy.sh/users/3377280)  
+*written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+What happens when a long note specialist joins forces with a metalhead? You get Euclid by Function Phantom (who is, by the way, one of osu! featured artists), a map both brutal and fascinating.
+
+A collab between juankristal and XeoStyle, this chart is a release heaven, which makes it both loved and hated. Its trademark patterns involve 3-note chords with each note being released separately as well as less dense sections which focus more on finger independency. Although not as hard to fullcombo as other long note oriented maps, Euclid is notably difficult to get a good accuracy on, especially because of a high OD setting (for a LN map). Because of its popularity and how well this chart is executed, we hope it will be well perceived in the Loved section!
+
+[![](/wiki/shared/news/2018-08-12-project-loved-week-of-august-12th/mania/ninja-re-bang-bang.jpg)](https://osu.ppy.sh/community/forums/topics/788689)
+
+[Kyary Pamyu Pamyu - Ninja Re Bang Bang](https://osu.ppy.sh/beatmapsets/372942) by [ecafree2](https://osu.ppy.sh/users/1984442)  
+*written by [TheToaphster](https://osu.ppy.sh/users/7616811)*
+
+Continuing on, we introduce ecafree2 to the Loved section with his chart for Ninja Re Bang Bang, a chart that has seemingly simple intentions at first glance, but hides a secret midway through that will certainly surprise anyone not expecting anything out of the ordinary. Transitioning out of the first chorus, the player is thrown into a 0.25x slow jam that for many players will have the notes too close together to effectively read, but ecafree compensates the player with how simple the rhythm and patterning is, allowing for an understanding of the section without too much learning. This novelty and the execution of it, surprising the player, as well as the popularity it has amassed for this reason gives it a good shot for the Loved section.
```

Comment:
typo with ecafree2's username?
```diff
- but ecafree compensates
+ but ecafree2 compensates
```

Reply:
I doubt that's a typo, most usernames are referred to without the numbers and things

Reply:
Alright, but I thought it wasn't that since the previous occurrence (in the first sentence) had it as `ecafree2`.

Reply:
that's true, fixed